syntax = "proto3";

package mathtest;

// Defines the service
service MathTest {
    // Function invoked to send the request
    rpc sendRequest (MathRequest) returns (MathReply) {}
    rpc sendarray (MathRequest) returns (Mathhasil) {}
}

// The request message containing requested numbers
message MathRequest {
    int32 a = 1;
    int32 b = 2;
}

// The response message containing response
message MathReply {
    int32 result = 1;
}

message Mathhasil {
    string header = 1;
    int32 fstop = 2;
    repeated int32 result_int = 3;
    repeated float result_float = 4;
    repeated double result_double = 5;
}
